test/time_test.py

```diff
@@ -4,6 +4,15 @@
 
 Clock = pygame.time.Clock
 
+SDL1 = pygame.get_sdl_version()[0] < 2
+
+# Because timing based tests are twitchy, just test stuff again if they fail,
+# but only upto NUM_TRIES
+NUM_TRIES = 3
+
```

Comment:
Could use an extra line break here to make linters happy with PEP8 compliance.

Reply:
Done :)